addons/core/addon/initializers/deprecations.js

```diff
@@ -7,6 +7,11 @@ import { registerDeprecationHandler } from '@ember/debug';
 
 export function initialize() {
   registerDeprecationHandler((message, options, next) => {
+    //silence deprecate-early-static warnings till we find a way to
```

Comment:
Maybe it would also help to add a TODO as part of the comment?

Reply:
done